edgelet/contrib/systemd/enterprise-linux/aziot-edged.service

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=Azure IoT Edge daemon
+After=network-online.target docker.service aziot-edged.workload.socket aziot-edged.mgmt.socket
+Requires=aziot-edged.workload.socket aziot-edged.mgmt.socket
+Wants=network-online.target docker.service
+Documentation=man:aziot-edged(8)
+
+[Service]
+ExecStart=/usr/libexec/aziot/aziot-edged
+KillMode=process
+TimeoutStartSec=600
+TimeoutStopSec=40
```

Comment:
I'm assuming we are not dealing with Pi the RedHat8.

Reply:
At the moment, the target is amd64 arch only.